rstb/rstb.py

```diff
@@ -193,12 +184,7 @@ def calculate_file_size_with_ext(self, file: typing.Union[str, os.PathLike, typi
         if wiiu:
             size += 0xe4 # res::ResourceMgr constant. Not sure what it is.
             size += info.size_wiiu
-            if actual_ext not in _factory_parsers:
-                size += info.parse_size_wiiu
-            elif info.is_complex:
-                size += _factory_parsers[actual_ext].parse_wiiu(file_data)
-            else:
-                size += 0
+            size += info.parse_size_wiiu
```

Comment:
The is_complex check needs to be kept, otherwise you'll be adding "complex" to `size` for complex resource classes.

Reply:
Don't the parse size values on `Factory` not even get set when a type is complex?
https://github.com/zeldamods/rstb/blob/1fdea29eecd7096a999de40f81d9b587959985a7/rstb/rstb.py#L144

Reply:
Oh, my bad, I forgot about that check.